src/pools/abstract-pool.ts

```diff
@@ -179,7 +189,7 @@ export abstract class AbstractPool<
         })
         return workerCreated
       },
-      opts.workerChoiceStrategy ?? WorkerChoiceStrategies.ROUND_ROBIN
+      opts?.workerChoiceStrategy ?? WorkerChoiceStrategies.ROUND_ROBIN
```

Comment:
If I see it correctly, we don't need this optional chain anymore :thinking:  
Because on line 175 we already access it without any check
```suggestion
      opts.workerChoiceStrategy ?? WorkerChoiceStrategies.ROUND_ROBIN
```